### PR TITLE
Feature/add nps

### PR DESCRIPTION
**Pull request by bobbytes**

### CODE REVIEW

**Review by hirsch88 — COMMENTED** [review]
Why do we need nps?

File: package-scripts.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+const { ifWindows } = require('nps-utils');
+
+/**
+ * Windows: Please do not use trailing comma as windows will fail with token error
+ */
+module.exports = {
+  scripts: {
+    default: 'build',
+    serve: ifWindows(
+      '.\\node_modules\\.bin\\vue-cli-service serve --open',
```
Comment (dweber019) [inline package-scripts.js]: You don't need any of this. npm handles windows and linux/mac already.

**Review by dweber019 — CHANGES_REQUESTED** [review]

File: package-scripts.js (hunk 1 above)
Comment (DaNautilus) [reply package-scripts.js]: appveyor does not find `./node_modules/.bin`

Comment (bobbytes) [reply package-scripts.js]: Done

**Review by dweber019 — APPROVED** [review]

**Pull request merged**